server/views/partials/layout.njk

```diff
@@ -32,5 +32,6 @@
   <body>, to avoid blocking the initial render. #}
   <script src="/assets/govuk/all.js"></script>
   <script src="/assets/govukFrontendInit.js"></script>
+  <script src="/assets/app.js"></script>
```

Comment:
Looking at the above file shouldn't the path be /assets/js/app.js ?

Reply:
That's a fair point. And yet it works for this and the other ones. There must be an alias or similar in place.

Reply:
Here it is: https://github.com/ministryofjustice/hmpps-digital-prison-reporting-mi-ui/blob/main/server/middleware/setUpStaticResources.ts#L41

Reply:
🤷‍♂️